### PR TITLE
Make source map generation optional

### PR DESCRIPTION
**Pull request by aldendaniels**
This is nice when debugging JSX code

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function reactify(filename, options) {
       try {
         var output = ReactTools.transform(source, {
           es5: options.target === 'es5',
-          sourceMap: true,
+          sourceMap: ('sourceMap' in options ? options.sourceMap : true),
```
Comment (andreypopp) [inline index.js]: I think it should handle the case then options are passed through command line:
```
% browserify -t [ reactify --source-map false ] ...
```
or
```
% browserify -t [ reactify --no-source-map ] ...
```

Comment (andreypopp) [inline index.js]: ``--no-source-map`` looks more useful, I think

**Comment by aldendaniels** [comment]
@andreypopp - I've made the change you suggested.

**Comment by hbrls** [comment]
@aldendaniels You've mentioned debugging. Did you encounter the same as I have? [The grey lines in Chrome and break points does not work](http://stackoverflow.com/questions/29650951/what-are-these-grey-lines-in-the-chrome-sources-panel)

**Comment by hbrls** [comment]
And if I change the `true` to `false` in the `node_modules/reactify/index.js`, the whole thing works as before and looks good to me.

**Comment by aldendaniels** [comment]
@hbrls - I hadn't noticed the gray bar issue with Reactify. 

I recently switch to [babelify](https://github.com/babel/babelify) - which also transpiles JSX -> JS.

Unlike Reactify, babelify uses [acorn-jsx](https://github.com/RReverser/acorn-jsx) under the hood instead of FBs offical (esprima-based) JSX transpiler.

Anyway, today I noticed that Babel has the same issue with gray bars that you described:

screen shot 2015-04-21 at 2 03 03 pm

Glad to know that disabling source maps fixes the issue.

**Comment by aldendaniels** [comment]
@hbrls - saw your comment in http://stackoverflow.com/questions/29650951/what-are-these-grey-lines-in-the-chrome-sources-panel

So you also noticed that the gray line issues occurs both on babelify and reactify.

**Pull request closed without merging**